fic_modules/git.py

```diff
@@ -546,15 +567,15 @@ def extract_json_from_git(json_files, path_to_files, days_to_generate):
         with open(file_path) as json_file:
             data = json.load(json_file)
             base_link = "https://github.com/mozilla-releng/firefox-infra-" \
-                        "changelog/blob/master/{}/"\
-                        .format(path_to_files)
-            repository_url = base_link + file\
-                .rstrip()\
-                .replace(" ", "%20")\
-                .rstrip()\
+                        "changelog/blob/master/{}/" \
+                .format(path_to_files)
```

Comment:
Indentation lost when compared to old revision.
Is it expected?

Reply:
No, it was not, probably hit auto-format (tho it doesn't affect the code or any lint), fixed in the next push.